src/common/fields-derive/src/lib.rs

```diff
@@ -117,7 +117,9 @@ fn get_primary_key(input: &syn::DeriveInput) -> Option<Vec<usize>> {
         return Some(
             keys.to_string()
                 .split(',')
-                .map(|s| index(s.trim()))
+                .map(|s| s.trim())
+                .filter(|s| !s.is_empty())
```

Comment:
So that we can attach `#[primary_key()]` on the struct to specify empty primary keys.